app/routes/codelists-management/index.js

```diff
@@ -13,16 +14,24 @@ export default class CodelistsManagementIndexRoute extends Route {
   };
 
   async model(params) {
+    const administrativeUnit = await this.store.findRecord(
+      'administrative-unit',
+      this.currentSession.group.id
```

Comment:
I'm not sure why we are requesting the unit, since we only use the id and already have that in this.currentSession.group.id

Reply:
You are right, to add it I have to look for the admin unit for some weird ember reason but not in this case